pubspec.lock

```diff
@@ -1645,8 +1645,8 @@ packages:
     dependency: "direct main"
     description:
       path: "."
-      ref: main
-      resolved-ref: "4a4ee913fbdec0f98cc1755f31df54fb402d0957"
+      ref: "bugfix/ios-editor-height"
```

Comment:
also merge and pump patch version for https://github.com/linagora/rich-text-composer.git

Reply:
Done